core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -161,6 +161,7 @@ private[spark] object UIUtils extends Logging {
 
   def commonHeaderNodes: Seq[Node] = {
     <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
+    <link rel="shortcut icon" href="about:blank"/>
```

Comment:
In the past week, I was trying to find out a solution for the overlaps but I didn't make it successfully, as I am not so familiar with CSS and frontend stuffs. I will try to find some professions in our company to solve this issue. 
Additionally, this line will disable the downloading for the favicon.icon.